proxies/live/apiproxy/policies/AssignMessage.AddCors.xml

```diff
@@ -4,9 +4,9 @@
     <Set>
         <Headers>
             <Header name="Access-Control-Allow-Origin">{request.header.origin}</Header>
-            <Header name="Access-Control-Allow-Headers">origin, x-requested-with, accept, content-type, nhsd-session-urid</Header>
+            <Header name="Access-Control-Allow-Headers">origin, x-requested-with, accept, content-type, if-match, nhsd-session-urid</Header>
             <Header name="Access-Control-Max-Age">3628800</Header>
-            <Header name="Access-Control-Allow-Methods">GET, PUT, POST, DELETE</Header>
+            <Header name="Access-Control-Allow-Methods">GET, PUT, POST, PATCH, DELETE</Header>
```

Comment:
Why did we add PATCH?

Reply:
Copied the proxies over from PDS. Will change to just GET and POST, as that's all we're using for the moment.

Reply:
Same goes for the if-match header, we're not using that (yet)